static/documentation/forking.md

```diff
@@ -0,0 +1,38 @@
+# Working with a forked repo
+
+## Forking the repo
+
+In order to contribute to this project you will need to create a fork of the repo on your own account. 
+To fork the repo just open the repository in a browser and click on the "fork" button on the top right of the page.
```

Comment:
Missing comma:
**To fork the repo, just open the repository in a browser and click on the "fork" button on the top right of the page.**

Reply:
Updated as suggested.